pkgs/development/python-modules/hikari-python/default.nix

```
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pytest-runner
+, aiohttp
+, attrs
+, multidict
+, colorlog
+, pynacl
+, git
+, pytest
+, pytest-cov
+, pytest-randomly
+, pytest-asyncio
+, mock
+  # Opts
+, enableRestServer ? false
+}:
+
+buildPythonPackage rec {
+  pname = "hikari-python";
```

Maybe name it
```suggestion
  pname = "hikari";
```
since it will already be called with `python3Packages.hikari`

---

I'm concerned about package name conflicts with the Hikari window manager (`pkgs.hikari`).

---

I've changed the name to plain `hikari` and it doesn't look like anything else broke... my bad!

---

Don't worry, nothing broke because it's when changing it you didn't change it to `pkgs.hikari` you changed it to `python3Packages.hikari` before it was `python3Packages.hikari-python`